scripts/docs.js

```diff
@@ -7,6 +7,7 @@ const nnsInputFiles = [
   "./packages/nns/src/genesis_token.canister.ts",
   "./packages/nns/src/governance.canister.ts",
   "./packages/nns/src/icp.ts",
+  "./packages/nns/src/token.ts",
```

Comment:
nice 👍

- can you add some docs to the class and its methods?

- speaking of docs, can you add an entry in CHANGELOG.md?

Reply:
How do I add the docs? I copied from ICP, I thought that they were there already as well.

Sure, I will add something in the changelog.

Reply:
I meant just normal docs

```
/**
 * This is a class
 * etc
 * @param etc.
 * etc
 **/
``` 

you are right seems ICP was not documented yet, it's a good occasion to improve this.

Reply:
sounds good

Reply:
Done